seleniumDescriptors.js

```diff
@@ -0,0 +1,177 @@
+var format = require('util').format;
+var util = require('./util');
+
+function Config(config) {
+	this.name = 'default';
+	if (config) {
```

Comment:
The `if (config)` check isn't needed since `util.mixin` will gracefully handle a `null` or `undefined` value of `config`.